src/ralph/back_office/models.py

```diff
@@ -58,6 +63,18 @@ def __str__(self):
     def __repr__(self):
         return '<BackOfficeAsset: {}>'.format(self.id)
 
+    def validate_imei(self):
+        is_imei = settings.IMEI_UNTIL_2003.match(self.imei) or \
+            settings.IMEI_SINCE_2003.match(self.imei)
```

Comment:
i'd revert this condition, to check first to match to new style imei - probably all of devices stored in ralph will have new style imei, so there would be no need to check second condition